File: IronVault/UI/ironvault_desktop/lib/models/faq.dart
import 'package:json_annotation/json_annotation.dart';


part 'faq.g.dart';

@JsonSerializable()
class Faq {
  int? faqid;
  String? pitanje;
  String? odgovor;


  Faq({this.faqid});

  factory Faq.fromJson(Map<String, dynamic> json) =>
      _$FaqFromJson(json);

  /// Connect the generated [_$PersonToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$FaqToJson(this);

}

File: IronVault/IronVault.Services/Database/Faq.cs
using System;
using System.Collections.Generic;

namespace IronVault.Services.Database;

public partial class Faq
{
    public int Faqid { get; set; }

    public string Pitanje { get; set; } = null!;

    public string Odgovor { get; set; } = null!;
}
